### ISSUE TITLE
`fs-extra` v10 is broken with `pkg`. 

### ISSUE THREAD

**Issue by jesec** [vercel/pkg#1180]
`fs-extra` released v10 and now it's broken with `pkg`. Here's the error I'm seeing:
```
TypeError: Cannot read property 'name' of undefined
    at /snapshot/project/node_modules/universalify/index.js:15:26
    at Object.<anonymous> (/snapshot/project/node_modules/fs-extra/lib/fs/index.js:57:27)
    at Module._compile (pkg/prelude/bootstrap.js:1503:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at Module.require (pkg/prelude/bootstrap.js:1409:31)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/snapshot/project/node_modules/fs-extra/lib/index.js:5:6)
```

Using `fs-extra@9` doesn't have this problem.

_Originally posted by @fcastilloec in https://github.com/vercel/pkg/issues/850#issuecomment-842816083_

**Comment by jesec** [comment]
cc: @robertsLando @erossignon

This is an urgent issue.

**Comment by jesec** [comment]
I took a second look. Apparently #1173 is related to `fs-extra`, but we haven't released a new version yet.

**Comment by jesec** [comment]
Can not reproduce on master branch. Presumptively already resolved by #1173.

@leerob We should release `5.2`, after we bump `pkg-fetch` to 3.1.0. 

Alternatively we can cherry-pick #1173 to `5.1.0`, and do a `5.1.1`.

**Comment by robertsLando** [comment]
Yeah I think that problem is already resolved by #1173
